tests/Fixtures/Integration/misc/PHP8_2.test

```diff
@@ -101,6 +101,12 @@ trait WithConstants
     private const THREE = 'three';
```

Comment:
should we also add this test?
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6629/files#diff-629ff8fa5f9f5ed321c8cfc177d41dd580f804921b576c1d023e9efafc4b5e03R1871

Reply:
done